package.json

```
@@ -197,4 +198,4 @@
     "prisma": "~2.20.0",
     "ts-node": "^9.1.1"
   }
-}
+}
```

```suggestion
}

```
always have a blank newline at the end of the file